src/org/thoughtcrime/securesms/ConversationAdapter.java

```diff
@@ -192,10 +202,14 @@ public void changeCursor(Cursor cursor) {
     super.changeCursor(cursor);
   }
 
+  void changeSearchTerm(String term) {
```

Comment:
do we want this as default (package) visibility?

Reply:
I had public at first but package makes more sense since it is only used in that scope.